app/helpers/katello/katello_urls_helper.rb

```diff
@@ -39,11 +39,10 @@ def subscription_manager_configuration_url(host = nil, rpm = true, hostname: nil
 
     apipie :method, 'Generates an absolute path to the file' do
       required :content_path, String, desc: "Relative path to the file or it's name"
-      optional :content_type, String, desc: 'Content type', default: 'repos'
       optional :schema, String, desc: 'Optional URL schema for the content source', default: 'http'
       returns String, desc: 'Absolute path to a file'
     end
-    def repository_url(content_path, content_type = 'repos', schema = 'http')
+    def repository_url(content_path, schema = 'http')
```

Comment:
this is a good change, but would require an update in a couple places in foreman:  

$ git grep repository_url\(
app/views/unattended/provisioning_templates/provision/kickstart_default.erb:    repository_url(img_name, 'isos')
app/views/unattended/provisioning_templates/provision/kickstart_ovirt.erb:  liveimg_url = repository_url(liveimg_name, 'isos')

I wonder if it would make sense to leave the variable and just not use it:  "_content_type = nil"  at least for a while

Reply:
I will open a follow on foreman PR later, after this is merged. But let's go with the default value as nil for content_type. I restored the optional parameter too until foreman has been changed.